config/app/setup/handlers/setup.json

```diff
@@ -10,6 +10,7 @@
       "args_errorHandler": { "@id": "urn:solid-server:default:ErrorHandler" },
       "args_responseWriter": { "@id": "urn:solid-server:default:ResponseWriter" },
       "args_operationHandler": {
+        "@id": "urn:solid-server:default:SetupHttpHandler_Specifics",
```

Comment:
Specifics? I don't understand 😅

Reply:
Can't we just drop that suffix?
Or look at the naming of the whole file; we have `urn:solid-server:default:SetupHttpHandler` that is a `ParsingHttpHandler`, and `args_operationHandler` that is a… `SetupHandler`. I'm very confused.

Reply:
I don't disagree about the naming. I think they were split to have some modularity for both accepting a setup request, and actually doing what is necessary for setup.

The parsing handler is called `SetupHttpHandler` so it has a similar identifier to all the other HTTP handlers in the list of those handlers. Chances are that originally this parsing handler was added later and the identifier was moved for consistency.